sys/vane/dill.hoon

```diff
@@ -507,9 +507,9 @@
       [~ ..^$]
     ::  this lets lib/helm send %heft a la |mass
     ::
-    =/  not=note-dill
-      ?:(?=([%crud %hax-heft ~] p.task) [%heft ~] p.task)
-    [[u.hey.all %slip %d not]~ ..^$]
+    =?  p.task  ?=([%crud %hax-heft ~] p.task)  [%heft ~]
+    ::
+    $(wrapped-task p.task)
```

Comment:
Does this `%=` need to also change `hen` to be `u.hey.all`?

Reply:
Yes, it does. I am somewhat embarrassed I missed that.